lib/menu.rb

```diff
@@ -0,0 +1,16 @@
+class Menu
+  attr_reader :dishes 
+  def initialize 
+    @dishes = [{ name: "pizza", price: 10.50 }, 
+      { name: "pasta", price: 9.45 },
+      { name: "calzone", price: 8.65 },
```

Comment:
Only one Calzone at Caz's Calzones? Not going to lie I am a bit disappointed :' (